core/build.sbt

```diff
@@ -0,0 +1,68 @@
+import Dependencies._
+
+scalariformSettings
+
+name := "pureconfig"
+
+organization := "com.github.melrief"
+
+version := "0.4.0"
+
+homepage := Some(url("https://github.com/melrief/pureconfig"))
+
+licenses := Seq("Mozilla Public License, version 2.0" -> url("https://www.mozilla.org/MPL/2.0/"))
+
+resolvers ++= Seq(
+  Resolver.sonatypeRepo("releases"),
+  Resolver.sonatypeRepo("snapshots")
+)
+
+libraryDependencies ++= Seq(
+  shapeless,
+  scalaMacrosParadise,
+  typesafeConfig,
+  scalaTest,
+  joda,
+  jodaConvert,
+  scalaCheck,
+  scalaCheckShapeless
+)
+
+initialize := {
```

Comment:
I believe this is redundant. I think `settings(settings)` in the root build applies the `initialize` in `settings`.